translations/ui-marc-authorities/en.json

```diff
@@ -92,5 +92,11 @@
   "settings.manageAuthoritySourceFiles.error.codes.unique": "Error saving data. Prefix must be unique.",
   "settings.manageAuthoritySourceFiles.error.codes.empty": "Error saving data. Prefix is required.",
   "settings.manageAuthoritySourceFiles.error.codes.length": "Error saving data. Prefix cannot be more than {maxlength} characters.",
-  "settings.manageAuthoritySourceFiles.create.success": "The authority file (name) has been successfully created."
+  "settings.manageAuthoritySourceFiles.create.success": "The authority file <b>{name}</b> has been successfully <b>created</b>.",
+  "settings.manageAuthoritySourceFiles.update.success": "The authority file <b>{name}</b> has been successfully <b>updated</b>.",
+  "settings.manageAuthoritySourceFiles.update.fail.fileAssigned": "Changes to <b>{name}</b> cannot be saved. Existing authority records are already assigned to this authority file.",
```

Comment:
It this message not used?

Reply:
Error messages are used here
https://github.com/folio-org/ui-marc-authorities/pull/358/files#diff-94dd02c0c33e0529538e0a59255d60351eb977c82e4971d252f38de317cca86dR56